### ISSUE TITLE
Proposal: add option to combine actions

### ISSUE THREAD

**Issue by nekromoff** [simonw/shot-scraper#124]
I believe it would be handy to both being able to screenshot a page and scrape some other data (e.g. title etc.) at the same time. 

Right now it is possible only by executing two command - one to take a screenshot, second to scrape element content via HTML or JS.

**Comment by simonw** [comment]
I agree, this is a good idea. Not sure how best to design this though - could be an option to `shot-scraper shot ...` or an option to `shot-scraper javascript ...`.
